Formula/b/basti.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class Basti < Formula
+  desc "Cloud credentials manager cli"
+  homepage "https://www.basti.app/"
+  url "https://registry.npmjs.org/basti/-/basti-1.6.2.tgz"
+  sha256 "8d813c1f4e3b8195655d40e670aa8a2eb7dce2cd21d996564c56a3296163f1d7"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Remove incompatible pre-built binary, session-manager-plugin
```

Comment:
For context, session-manager-plugin doesn't support arm64 architecture so I'm using amd64 for this platform since, as far as I know, it's compatible with Apple Silicon (at least, my coworkers use Basti on their M1 Macbooks). 

Maybe it's still better to leave this binary or add session-manager-plugin as a dependency of this Formula?

Reply:
mach-o format is kind of wrong.

----

can you point me about the source repo of it, is this the one? https://github.com/aws/session-manager-plugin